src/Timers.cpp

```diff
@@ -300,14 +540,70 @@ void Timers::GetTimerTypes(std::vector<PVR_TIMER_TYPE> &types)
       PVR_TIMER_TYPE_SUPPORTS_WEEKDAYS         |
       PVR_TIMER_TYPE_SUPPORTS_START_END_MARGIN,
       ""); /* Let Kodi generate the description */
+  types.push_back(*t);
```

Comment:
Did you know that `emplace_back` is to prefer over `push_back` as the former does not make unneded copies of the elements?